src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -109,147 +86,69 @@ WMTS_Provider.prototype.url = function url(coWMTS, layer) {
  * @param {type} coWMTS : coord WMTS
  * @returns {WMTS_Provider_L15.WMTS_Provider.prototype@pro;_IoDriver@call;read@call;then}
  */
-WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, parameters) {
-    var cooWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
-    var pitch = new THREE.Vector3(0.0, 0.0, 1.0);
-
-    if (parameters.ancestor) {
-        // account for possible level offset between coords and tile level
-        // (e.g for PM texture cooWMTS.level = tile.level + 1)
-        var levelOffset = cooWMTS.zoom - tile.level;
-
-        cooWMTS = this.projection.WMTS_WGS84Parent(
-            cooWMTS,
-            this.computeLevelToDownload(tile, parameters.ancestor, layer) + levelOffset,
+WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, parentTextures) {
+    let coordWMTS = tile.wmtsCoords[layer.options.tileMatrixSet][0];
+    const pitch = new THREE.Vector3(0.0, 0.0, 1.0);
+
+    if (parentTextures) {
+        coordWMTS = WMTS_WGS84Parent(
+            coordWMTS,
+            parentTextures[0].coordWMTS.zoom,
             pitch);
+        const { min, max } = OGCWebServiceHelper.ioDXBIL.computeMinMaxElevation(
+                parentTextures[0].image.data,
+                SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
+                pitch);
+        return Promise.resolve({ pitch, texture: parentTextures[0], min, max });
```

Comment:
I'd suggest to *not* use `Promise` where not needed.
Just return your object if this task can be performed directly.
You'll to adapt `Scheduler` line:
```
var p = provider.executeCommand(cmd) || Promise.resolve();
```
to test if the result from `executeCommand` is a Promise or a plain object.